nixos/modules/services/networking/bee.nix

```diff
@@ -103,6 +110,8 @@ After you finish configuration run 'sudo bee-get-addr'."
       description = "Ethereum Swarm Bee daemon user";
       home = cfg.settings.data-dir;
       isSystemUser = true;
+      extraGroups = optional config.services.bee-clef.enable
+        config.services.bee-clef.group;
```

Comment:
@SuperSandro2000 is this doing what i think it does? when bee-clef is enabled, then it takes the configured bee-clef group and adds the bee user into that group?

Reply:
Yes.